dev_guide/app_tutorials/maven_tutorial.adoc

```diff
@@ -78,6 +78,11 @@ $ oc set probe dc/nexus \
 	--failure-threshold 3 \
 	--initial-delay-seconds 30 \
 	-- echo ok
+oc set probe dc/nexus \
+	--readiness \
+	--failure-threshold 3 \
+	--initial-delay-seconds 30 \
+	--get-url=http://:8081/content/groups/public
```

Comment:
it depends, but in this case if it's not serving content, something has gone wrong and you probably want it restarted, so i'd expect them to be the same.